README.md

```diff
@@ -96,15 +96,49 @@ pip install -e ".[test,dev]"
 pip install -e plugins/fuzzyhashes
 ```
 
+## Settings
+
+Surfactant settings can be changed using the `surfactant config` subcommand, or by hand editing the settings configuration file (this is not the same as the JSON file used to configure settings for a particular sample that is described later).
+
+### Command Line
+
+Using `surfactant config` is very similar to basic use of `git config`. The key whose value is being accessed will be in the form `section.option` where `section` is typically a plugin name or `core`, and `option` is the option to set. As an example, a plugin that includes an entire copy of the original files as part of the generated SBOM might provide an option to specify the format.
+
+Setting the configuration option to `base64` could be done using:
+
+```bash
+surfactant config include_entire_file.format base64
+```
+
+Getting the currently set value for the option would then be done with:
+
+```bash
+surfactant config include_entire_file.format
```

Comment:
Is this a working example or does this setting not exist?

Reply:
The setting doesn't exist, so it won't actually affect how Surfactant runs. It works in the sense that setting and getting the value will save the value to the underlying config file and retrieve the stored value.

Reply:
I might update this to use setting the recorded institution as an example (along with making that one of the first config options available).

Reply:
This has been updated to be the recorded_institution setting now.